--- package/lean/default-settings/files/zzz-default-settings	2021-04-02 23:54:56.972096864 +0800
+++ package/lean/default-settings/files/zzz-default-settings.aa	2021-04-02 23:56:16.235974370 +0800
@@ -10,6 +10,12 @@
 uci set fstab.@global[0].anon_mount=1
 uci commit fstab
 
+uci set network.lan.ipaddr=192.168.233.116
+uci set network.lan.gateway=192.168.233.1
+uci set network.lan.dns=192.168.233.116
+uci commit network
+ifup lan
+
 rm -f /usr/lib/lua/luci/view/admin_status/index/mwan.htm
 rm -f /usr/lib/lua/luci/view/admin_status/index/upnp.htm
 rm -f /usr/lib/lua/luci/view/admin_status/index/ddns.htm
